store/test-store/src/lib.rs

```diff
@@ -29,12 +29,11 @@ lazy_static! {
         None => Logger::root(slog::Discard, o!()),
     };
 
-    // Use this for tests that need subscriptions from `STORE`.
-    pub static ref STORE_RUNTIME: Mutex<Runtime> = Mutex::new(Runtime::new().unwrap());
+    pub static ref STORE_RUNTIME: Mutex<Runtime> = Mutex::new(Builder::new().basic_scheduler().enable_all().build().unwrap());
 
     // Create Store instance once for use with each of the tests.
     pub static ref STORE: Arc<Store> = {
-        STORE_RUNTIME.lock().unwrap().block_on(future::lazy(|| -> Result<_, ()> {
+        STORE_RUNTIME.lock().unwrap().block_on(async {
```

Comment:
There's a reason for this, I think it's because this will use `spawn` internally so it needs to be in the context of a runtime. The way this is setup is not the best but works for now.